README.md

```diff
@@ -13,6 +13,12 @@ PAYPAL_API_BASE_URL=https://api-m.sandbox.paypal.com
 PAYPAL_WEB_BASE_URL=https://www.paypal.com
 ```
 
+This application requires Node.js version 16.8.0 or higher. Ensure that the correct Node.js version is being used by running:
```

Comment:
```suggestion
This application requires Node.js version 16.8.0 or higher. 
```

Could we remove the `nvm` command? Technically there are other version managers so I don't want to assume everyone is using nvm. For example, I like using [volta](https://docs.volta.sh/guide/)

Reply:
Ah good point!  Done.